census/static/scripts/form.js

```diff
@@ -38,4 +38,18 @@ $( document ).ready(function() {
     $( this ).parents(".question").addClass( "answered" );
   });
 
+
+  // heading dropdown
+  $(".form-header .primary-meta h1").each(function() {
```

Comment:
This is from @smth, originally. It's not part of the React controlled area of the page.